### PR TITLE
docs(cn): update editor setup

### PR DESCRIPTION
**Pull request by kecrily**
#### What kind of changes does this PR include?

- Translated content

#### Description

update editor setup

### CODE REVIEW

File: src/pages/zh-cn/editor-setup.md [hunk 1]
```diff
@@ -20,19 +20,62 @@ Astro 可以和任意编辑器一同工作。不过我们推荐在 Astro 项目
...
+
+## 其他工具
+
+### ESLint
+
+[ESLint](https://eslint.org/) 是流行的 JavaScript 和 JSX 的 linter。为了支持 Astro，需要安装[一个由社区维护的插件](https://github.com/ota-meshi/eslint-plugin-astro)。
+
+关于如何为你的项目安装和设置 ESLint 的更多信息，请参见[the project's User Guide](https://ota-meshi.github.io/eslint-plugin-astro/user-guide/)。
+
+### ＃＃＃Prettier
```
Comment (Yan-Thomas) [inline src/pages/zh-cn/editor-setup.md:55]:
```suggestion
### Prettier
```

File: src/pages/zh-cn/editor-setup.md [hunk 2]
```diff
@@ -20,19 +20,62 @@ Astro 可以和任意编辑器一同工作。不过我们推荐在 Astro 项目
...
+
+然后 Prettier 会自动检测到这个插件，并在运行时使用它来处理 `.astro` 文件；
+
+``shell
+prettier --write .
+```
+
+参见 [Prettier 插件的 README](https://github.com/withastro/prettier-plugin-astro/blob/main/README.md)，以获得更多关于其支持的选项、如何在 VS Code 内设置 Prettier 等信息。
+
+::caution[与 pnpm 一起使用]
```
Comment (Yan-Thomas) [inline src/pages/zh-cn/editor-setup.md:73]:
```suggestion
:::caution[与 pnpm 一起使用]
```

File: src/pages/zh-cn/editor-setup.md [hunk 3]
```diff
@@ -20,19 +20,62 @@ Astro 可以和任意编辑器一同工作。不过我们推荐在 Astro 项目
...
-- [Gitpod](https://gitpod.io) - 云上的完整开发环境，可以从 Open VSX 上安装官方 Astro VS Code 扩展。 
+- [Gitpod](https://gitpod.io) - 云上的完整开发环境，可以从 Open VSX 上安装官方 Astro VS Code 扩展。
+
+## 其他工具
+
+### ESLint
+
+[ESLint](https://eslint.org/) 是流行的 JavaScript 和 JSX 的 linter。为了支持 Astro，需要安装[一个由社区维护的插件](https://github.com/ota-meshi/eslint-plugin-astro)。
+
+关于如何为你的项目安装和设置 ESLint 的更多信息，请参见[the project's User Guide](https://ota-meshi.github.io/eslint-plugin-astro/user-guide/)。
```
Comment (Yan-Thomas) [inline src/pages/zh-cn/editor-setup.md:53]: There's some leftover English text here

File: src/pages/zh-cn/editor-setup.md [hunk 4]
```diff
@@ -20,19 +20,62 @@ Astro 可以和任意编辑器一同工作。不过我们推荐在 Astro 项目
...
+
+[Prettier](https://prettier.io/) 是流行的 JavaScript、HTML、CSS等 的格式化器。为了增加对 `.astro` 文件格式化的支持，请使用[官方 Astro Prettier 插件](https://github.com/withastro/prettier-plugin-astro)。
+
+要开始使用，首先要安装 Prettier 和该插件：
+
+``shell
+npm install --save-dev prettier prettier-plugin-astro
+```
+
+然后 Prettier 会自动检测到这个插件，并在运行时使用它来处理 `.astro` 文件；
```
Comment (Yan-Thomas) [inline src/pages/zh-cn/editor-setup.md:65]:
```suggestion
然后 Prettier 会自动检测到这个插件，并在运行时使用它来处理 `.astro` 文件：
```

**Review by Yan-Thomas — COMMENTED** [review]
Thanks for this @kecrily, I pointed out some errors and added suggestions, after these, LGTM!

File: src/pages/zh-cn/editor-setup.md (hunk 3 above)
Comment (kecrily) [reply src/pages/zh-cn/editor-setup.md:53]: This is deliberate, as the original title is in English.

**Comment by Yan-Thomas** [comment]
LGTM. Thanks @kecrily!

**Pull request merged**